src/util.c

```diff
@@ -206,6 +206,16 @@ int verify_args(struct ntttcp_test *test)
 		test->parallel = MAX_NUM_THREADS;
 	}
 
+	if (test->conn_per_thread < 1) {
+		PRINT_INFO("You need at least one connection per thread. Using one connection per thread.");
```

Comment:
I used bad name of "conn_per_thread". The meaning should be "connections per server port".
So the message could be ""invalid connections-per-server-port provided. use 1"